portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase

```diff
@@ -22,6 +22,22 @@ definition {
 		}
 	}
 
+	@description = "Verify remote app can be created"
+	@priority = "5"
+	test CanBeCreated {
```

Comment:
This is from IFrameCanBeCreated right?

Reply:
Yes.  Same as before, there were some changes in this test case that didn't appear in my conflicts.  I have changed it now.